src/Mutator/Removal/FunctionCallRemoval.php

```diff
@@ -48,6 +48,18 @@ final class FunctionCallRemoval implements Mutator
 {
     use GetMutatorName;
 
+    /** @var string[] */
+    private $whitelisted = [
```

Comment:
whitelist probably is the name best to be avoided; say, it can be `doNotRemoveList` instead

Reply:
Updated the names and the PR name